jgrapht-core/src/test/java/org/jgrapht/alg/scoring/AlphaCentralityTest.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * (C) Copyright 2018-2018, by Pratik Tibrewal and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg.scoring;
+
+import org.jgrapht.alg.interfaces.*;
+import org.jgrapht.graph.*;
+
+import junit.framework.*;
+
+/**
+ * Unit tests for AlphaCentrality
+ * 
+ * @author Pratik Tibrewal
+ */
+public class AlphaCentralityTest
```

Comment:
We recently got rid of all of the old JUnit3 test conventions and replaced them with JUnit4.  So for new code:

1) do use the `@Test` annotation

2) do NOT extend `TestCase`